player/lua/osc.lua

```diff
@@ -2562,7 +2562,7 @@ end
 mp.register_event("shutdown", shutdown)
 mp.register_event("start-file", request_init)
 mp.observe_property("track-list", "native", request_init)
-mp.observe_property("playlist", "native", request_init)
+mp.observe_property("playlist-count", "native", request_init)
```

Comment:
These were changed to "native" in https://github.com/mpv-player/mpv/commit/de0849404b65876f0f8fe934c4b74f497bf4e29b to avoid encouraging people to imitate the code, but I guess you're right, both can be nil.

Reply:
We changed to native in de0849404b65876f0f8fe934c4b74f497bf4e29b